components/Button/Button.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Link from 'next/link';
+import ButtonProps from './interfaces';
+import { BackButton } from './styles';
+
+export const Button = ({ text }:ButtonProps) => (
+  <Link href="/">
+    <BackButton variant="contained"> { text } </BackButton>
+  </Link>
+);

```

Comment:
Es macht nicht so viel Sinn, die href fest zu definieren, weil dann die button nicht wiederverwendbar wäre 
```suggestion
export const Button = ({ text, link }:ButtonProps) => (
  <Link href={link}>
    <BackButton variant="contained"> { text } </BackButton>
  </Link>
);

```